kolibri/core/assets/src/views/tab-link/index.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+
+  <li>
+    <router-link
+      :to="link"
+      class="tab"
+      :class="{ selected: selected, 'tab-icon-title': icon && title }">
```

Comment:
In react-router, there is an `activeClass` prop, which lets you specify a class to use when you're at the `to` address, instead of doing the `{ selected: selected }` thing here.

Does vue-router provide something similar?